src/views/ManageList.jsx

```diff
@@ -63,9 +81,8 @@ export function ManageList() {
 					<br />
 					<input
 						type="text"
-						defaultValue=""
 						id="item-name"
-						onChange={handleTextChange}
+						onChange={setItemName}
```

Comment:
Should this perhaps be using the new unified `handleTextChange` funtion? Unfortunately, the current implementation introduces a breaking change to the existing functionality, as it shows `[object Object]` to the user if they try to type anything in this field.

Reply:
yes, you are right, I must have edited this by mistake, thank you for catching that!